Makefile

```diff
@@ -1,4 +1,42 @@
-.PHONY: build check deps deps-macos clean compile_starknet clippy remove-venv venv-test test clean coverage
+.PHONY: build check clean clippy compile-cairo compile-starknet coverage deps deps-macos remove-venv test
+
+
+OS := $(shell uname)
+ifeq ($(OS), Darwin)
+	CFLAGS  += -I/opt/homebrew/opt/gmp/include
+	LDFLAGS += -L/opt/homebrew/opt/gmp/lib
+endif
+
+
+CAIRO_SOURCES=$(wildcard cairo_programs/*.cairo)
+CAIRO_TARGETS=$(patsubst %.cairo,%.json,$(CAIRO_SOURCES))
+
+STARKNET_SOURCES=$(wildcard tests/*.cairo)
```

Comment:
Shouldn't we include starknet_programs here too?
@juanbono @azteca1998

Reply:
No, since they've been all moved into `tests/` because they're used in integration testing.